### ISSUE TITLE
EU High Value Datasets

### ISSUE THREAD

**Issue by StephanGeorg** [openaddresses/openaddresses#7170]
It's happening: https://inspire-geoportal.ec.europa.eu/srv/eng/catalog.search#/hvdshome

> *High-Value Datasets*
> Application providing an overview and accessing to geospatial high-value datasets and other core datasets (including priority datasets for eReporting), which fall under the scope of the [Open Data Directive](https://eur-lex.europa.eu/eli/dir/2019/1024/oj) according to the related [Implementing Act](https://eur-lex.europa.eu/eli/reg_impl/2023/138/oj).

#7055 

FYI @sabas @justinelliotmeyers @alesarrett @aborruso

**Comment by aborruso** [comment]
Thank you very much @StephanGeorg

**Comment by aborruso** [comment]
> Oh, it does list previous datasets only at the moment :(

Yes, unfortunately, until the deadline comes, I'm afraid there won't be much. At least in Italy.

**Comment by sergiy-theo** [comment]
Sweden promises to have it by next February: https://www.lantmateriet.se/en/geodata/geodata-products/open-data/high-value-data-hvd/

**Comment by justinelliotmeyers** [comment]
Sweden, Portugal, Italy and Bavaria are the outliers. 
Sweden funding approval happened but they will not share freely until they receive the funds.
Portugal is a mess. Still building data. Its somewhat challenging to speak to anyone in their government.
Italy - deadline and hopefully data starts flowing.
Bavaria will continue to charge - most likely no change anytime soon.

**Comment by StephanGeorg** [comment]
Yes, Bavaria even sued somebody (https://netzpolitik.org/2021/offene-geodaten-bayern-geht-gegen-open-data-aktivistinnen-vor/) for using the data. It's a mess and Bavaria (https://www.byte.bayern/) should shame for pretending being innovative in that regard.

**Comment by justinelliotmeyers** [comment]
It isn't in byte's control. It's up to the actual agency/ government. byte is different, they just host the data. they want the data to be open (from the two people I video chatted with two months ago), not sure about upper level managers/ leaders however.

**Comment by justinelliotmeyers** [comment]
AL,Shqipëria,Shqipëria,Albania
ad- NO DOWNLOAD - https://geoportal.asig.gov.al/map/?themeId=3303812&auto=true

Andorra
ad-https://www.ideandorra.ad/geoportal/index.jsp?lang=ca

CZ,Česko,Česko,Czechia
ad-https://services.cuzk.cz/gml/inspire/ad/epsg-5514/

DE,Deutschland,Deutschland,Germany
ad-**individual admins

DK,Danmark,Danmark,Denmark
ad-https://www.geodata-info.dk/srv/eng/catalog.search#/metadata/50b921ea-935e-d605-2287-4ee364046795

CY,Kýpros,Κύπρος,Cyprus
ad- NO DOWNLOAD - 

BE,Belgique/België,Belgique/België,Belgium
ad-https://www.geo.be/catalog/details/ca0fd5c0-8146-11e9-9012-482ae30f98d9?l=en

BG,Bulgaria,България,Bulgaria
ad- NO DOWNLOAD - 

CH,Schweiz/Suisse/Svizzera,Schweiz/Suisse/Svizzera,Switzerland
ad-swisstopo.admin.ch/en/geodata/official-geographic-directories/building-addresses.html

AT,Österreich,Österreich,Austria
ad-https://www.bev.gv.at/Services/Downloads/Produktbezogene-Downloads/Unentgeltliche-Produkte/Adressregister.html

EE,Eesti,Eesti,Estonia
ad-https://geoportaal.maaamet.ee/eng/Spatial-Data/Address-Data-p313.html

EL,Elláda,Ελλάδα,Greece
ad- NO DOWNLOAD - 

ES,España,España,Spain
ad-https://www.catastro.minhap.es/webinspire/index.html

FI,Suomi/Finland,Suomi/Finland,Finland
ad-https://www.avoindata.fi/data/fi/dataset/postcodes

FR,France,France,France
ad-https://adresse.data.gouv.fr/ / https://geoservices.ign.fr/ban-plus

HR,Hrvatska,Hrvatska,Croatia
ad-https://geoportal.nipp.hr/geonetwork/srv/eng/catalog.search#/metadata/d3104048-688a-4305-9471-77288cc6e350

HU,Magyarország,Magyarország,Hungary
ad- NO DOWNLOAD - 

IE,Éire/Ireland,Éire/Ireland,Ireland
ad- NO DOWNLOAD - 

IS,Ísland,Ísland,Iceland
ad-https://fasteignaskra.is/gogn/grunngogn-til-nidurhals/stadfangaskra/

IT,Italia,Italia,Italy
ad- NO DOWNLOAD - 

LV,Latvija,Latvija,Latvia
ad-https://data.gov.lv/dati/lv/dataset/valsts-adresu-registra-informacijas-sistemas-atvertie-dati

ME,Crna Gora,Црна Гора,Montenegro
ad- NO DOWNLOAD - 

MT,Malta,Malta,Malta
ad- NO DOWNLOAD - 

NO,Norge,Norge,Norway
ad-https://kartkatalog.geonorge.no/metadata/inspire-addresses/8a3da298-dad8-4b8f-b8fc-668efffad0d7

MK,Severna Makedonija,Северна Македонија,North Macedonia
ad- NO DOWNLOAD - 

LT,Lietuva,Lietuva,Lithuania
ad-https://www.registrucentras.lt/p/1187

LU,Luxembourg,Luxembourg,Luxembourg
ad-https://data.public.lu/en/datasets/adresses-georeferencees-bd-adresses/

NL,Nederland,Nederland,Netherlands
ad-https://www.nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/a5f961e9-ebdd-41e2-b8e8-

SI,Slovenija,Slovenija,Slovenia
ad-https://ipi.eprostor.gov.si/jgp/data

LI,Liechtenstein,Liechtenstein,Liechtenstein
ad-https://service.geo.llv.li/download/

SE,Sverige,Sverige,Sweden
ad- NO DOWNLOAD - 

PL,Polska,Polska,Poland
ad-https://opendata.geoportal.gov.pl/prg/adresy/PRG-punkty_adresowe.zip

SK,Slovensko,Slovensko,Slovakia
ad- NO DOWNLOAD

RS,Serbia,Srbija/Сpбија,Republic of Serbia
ad- NO DOWNLOAD - 

TR,Türkiye,Türkiye,Turkey
ad- NO DOWNLOAD - 

UK,United Kingdom,United Kingdom,United Kingdom
ad-NO OPEN DATA

RO,România,România,Romania
ad- NO DOWNLOAD - https://renns.ancpi.ro/renns-public/

PT,Portugal,Portugal,Portugal
ad-http://inspire.ine.pt/AD/atom/downloadservice.xml (**RESIDENTIAL ONLY?)

**Comment by justinelliotmeyers** [comment]
In theory these should all be split into their own unique tickets and see if they update current data, and counts should loosely align. Thoughts @iandees

**Comment by iandees** [comment]
Thanks for collecting these, @justinelliotmeyers. I agree that these should get their own tickets to track progress more easily.

I can try to do that this weekend but feel free to do it too!

**Comment by StephanGeorg** [comment]
What about

SK,Slovensko,Slovensko,Slovakia
ad- https://data.slovensko.sk/datasety/b27f57f1-7e76-45e0-8968-631f9176b2e9

**Comment by jwass** [comment]
@justinelliotmeyers I missed your comment here until now.

```
Also, @jwass I don't know where your teams sit at with doing anything for addresses, but this is a nice start. I've been asking overture/ amy/ marc about this, but nobody is saying anything.
```

Is the plan for that big list you posted to bring those into OpenAddresses?

**Comment by justinelliotmeyers** [comment]
@jwass So yes, ideally we want to update all sources with new data, and clear copyright. The above list all have a friendly copyright. The Poland one is somewhat tricky to find from what I recall, but otherwise all open and friendly.

UK - we can do UPRN codes, but 0 situs address data, unless we link from a secondary dataset for partial joins and only use for climate or something weird, but basically no derivatives.

We need to split these out into new tickets and then determine what we have under the hood now if it is same/ similar/ determine fields and then pr/ ingest.

I know overture pulled a lot of their v0 or v1 data from OA. It would be great to see people dedicated to spending time in here to do it (unless that is you, which I'm guessing it is!!). Once more people participate, momentum occurs and others participate (in theory). I need to spend more time in here for sure, so does Ian! But we all do - so however I can help, let me know and I can try to dedicate some time. I try to help where I can.

Have a great weekend

**Comment by jwass** [comment]
> I know overture pulled a lot of their v0 or v1 data from OA. It would be great to see people dedicated to spending time in here to do it (unless that is you, which I'm guessing it is!!). Once more people participate, momentum occurs and others participate (in theory). I need to spend more time in here for sure, so does Ian! But we all do - so however I can help, let me know and I can try to dedicate some time. I try to help where I can.

Yeah - we've been directing people here. It's not just me but also folks from AddressForAll and Esri and maybe others have been putting up PRs. The only non-OA source we used was the US NAD since it's not yet here: https://github.com/openaddresses/openaddresses/pull/7253

**Comment by iandees** [comment]
Hey all! Good to see discussion happening here. I don't have the capacity to help very much on the outreach to get this data, but I can definitely help create sources and conform JSON if we have download links. I'll try to poke through the links above and see if I can update some data sources this weekend. Feel free to point me directly at download links, too!
